test/integration/docker-compose-migration.yml.template

```diff
@@ -6,7 +6,7 @@ services:
     image: {{ .DockerImage }}
     ports:
       - "{{with .ExposedRPCPorts}}{{index . 0}}:{{end}}8484"
-      - "{{with .ExposedRPCPorts}}{{index . 0}}:{{end}}8485"
+      - "1{{with .ExposedRPCPorts}}{{index . 0}}:{{end}}8485"
```

Comment:
So both docker 8484 and 8485 is mapped to same host port, this won't start on local machine due to port collision. I don't know why this would work on CI.

Reply:
In the other compose files , I do stuff like `{{index . 0 | plus 123}}` to tweak this.  But yeah, I think something strange was going on on docker / CI such that it didn't fail just starting the containers like this

Reply:
Ah! I think CI passes only because we don't use integration.WithExposedRPCPorts() on any test-cases other than in  task dev:testnet:up. In all other testcases, the exposed ports are different.